stat/distuv/lognormal_test.go

```diff
@@ -42,3 +42,15 @@ func TestLognormal(t *testing.T) {
 		checkProbQuantContinuous(t, i, x, f, tol)
 	}
 }
+
+func TestLognormalCDFPrecision(t *testing.T) {
```

Comment:
Could you rename to `TestLognormalIssue577`?

Reply:
And please add a link to the issue.